packages/typespec-ts/src/modular/modularCodeModel.ts

```diff
@@ -124,6 +130,7 @@ export interface Client {
   rlcClientName: string;
   subfolder?: string;
   rlcHelperDetails: HelperFunctionDetails;
+  dpgClient: SdkClient;
```

Comment:
just curious why we need to store the SDKClient or SdkType? feel like we should be able to get everything in buildCodeModel.ts?

Reply:
The refactor is consuming TCGC types directly now, so these are being used as a key for that purpose.

Reply:
FYI,  we will have a refactor to fully adopt to tcgc SDKTypes https://github.com/Azure/autorest.typescript/issues/2410, I see, we are still using _raw in some places, but I think in this PR this is good for now.  and if you are interested, maybe you can work on that later ? :)